accelerator/common_core.c

```diff
@@ -4,50 +4,72 @@
 
 int cclient_get_txn_to_approve(const iota_client_service_t* const service,
                                ta_get_tips_res_t* res) {
+  if (res == NULL) {
```

Comment:
Can you denote the return code listing in core implementation? We shall have conventions in common.

Reply:
Two situation here:
* If the function has return value (mostly from entangled), return it directly
* Return `-1` for other checks
    * Check if arguments are valid before execution
    * Check if objects are valid after `xxx_new()`

Reply:
Send another pull request to append return code information for public functions, then can be generated into API Reference Manual later.

Reply:
Do you mean descriptions for each function like the below link?
https://github.com/iotaledger/entangled/blob/eceb7e5604455173e486884cbc79e887e93aa7b3/cclient/iota_client_core_api.h#L38

Reply:
Exactly.